templates/taxi/driver_list.html

```diff
@@ -27,4 +30,7 @@
     {% else %}
       <p>There are no drivers in the service.</p>
     {% endif %}
+   <div class="d-flex justify-content-start" style="margin-bottom: 10px">
+      <a href="javascript:history.back()" class="btn btn-secondary">Return</a>
```

Comment:
In terms of code style, you should use 2 whitespaces indentations in your `.html` files. It seems like you are using 4 spaces for indentation on this line. Please adjust it to use 2 spaces.